pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -205,6 +205,9 @@ func (b *bucket) aggregateCounts(p pb.ClientStatsPayload) {
 	}
 	for _, s := range p.Stats {
 		for _, sb := range s.Stats {
+			if sb == nil {
+				sb = &pb.ClientGroupedStats{}
+			}
```

Comment:
Isn't this check already happening? https://github.com/DataDog/datadog-agent/blob/ec8f26cc45b1376f8a3e04e2916f194b10bf215d/pkg/trace/stats/client_stats_aggregator.go#L270